doc/sphinx/using/libraries/funind.rst

```diff
@@ -1,6 +1,18 @@
 Functional induction
 ====================
 
+.. note::
+
+   The functional induction (FunInd) plugin is considered legacy. For new
+   code and new projects, the recommended replacement is `Equations
+   <http://mattam82.github.io/Coq-Equations/>`_, a more powerful plugin
+   available through the Coq Platform, which provides most features from
+   FunInd as well as several others. Refer to the `Equations documentation
+   <https://raw.githubusercontent.com/mattam82/Coq-Equations/master/doc/equations.pdf>`_
+   to learn more. FunInd is not deprecated and not planned for removal
+   yet because porting code from FunInd to Equations can be difficult
```

Comment:
```suggestion
   to learn more. FunInd is not deprecated and has not been removed
   yet because porting code from FunInd to Equations can be difficult
```

Reply:
I think I will keep the previous version, because with your suggestion, it could alert users that it could be removed at any point in the future, which is not the case.

Reply:
Sounds like we have no specific plan (eg a target date, ways to make the migration easier) and may never have one.

One could also remove the "yet", but enough with the wordsmithing.

Reply:
There is no specific plan, but we know that we want to get rid of this code eventually.